src/reducers/getPOI.reducer.js

```diff
@@ -0,0 +1,34 @@
+import initialState from './initialState'
+import {
+  LOAD_POIS,
+  LOAD_POI_ID,
+  LOAD_POI_MAP_YEAR,
+  LOAD_POI_STORY_ID
+} from './../actions/actionTypes'
+
+export default function getPoi(state = initialState, action) {

```

Comment:
the `state` param should default to the "slice" that this particular reducer updates. in this case, it would be `initialState.pois`